extensions/amp-ooyala-player/0.1/amp-ooyala-player.js

```diff
@@ -109,7 +109,7 @@ class AmpOoyalaPlayer extends AMP.BaseElement {
     const playerVersion = el.getAttribute('data-playerversion') || '';
     if (playerVersion.toLowerCase() == 'v4') {
       src =
-        'https://player.ooyala.com/static/v4/sandbox/amp_iframe/' +
+        'https://player.ooyala.com/static/v4/production/' +
```

Comment:
You need to update the tests in `amp-ooyala-player/0.1/tests` to reflect the new URL :) Also please make sure the API stays the same, if there are changes we might wanna consider releasing a `0.2` version instead.

Reply:
Thank you for the update! Just made the update in `amp-ooyala-player/0.1/tests`. The API remains the same and the query parameters will be allowed in the production latest version too.

Reply:
Hey! I’m still only seeing one changed file. Could you please make sure you pushed?

Reply:
Thank you for the heads up! This is done now :)

Reply:
Sorry one more thing, we might need to update the test video since the URL [https://player.ooyala.com/static/v4/production/latest/skin-plugin/amp_iframe.html?pcode=5zb2wxOlZcNCe_HVT3a6cawW298X&ec=Vxc2k0MDE6Y_C7J5podo3UDxlFxGaZrQ&pbid=6440813504804d76ba35c8c787a4b33c](https://player.ooyala.com/static/v4/production/latest/skin-plugin/amp_iframe.html?pcode=5zb2wxOlZcNCe_HVT3a6cawW298X&ec=Vxc2k0MDE6Y_C7J5podo3UDxlFxGaZrQ&pbid=6440813504804d76ba35c8c787a4b33c) seems to be no longer valid

Reply:
No worries! This can be updated at `validator-amp-ooyala.html`, correct?